openslides_backend/presenter/check_database.py

```diff
@@ -39,40 +78,35 @@ def get_result(self) -> Any:
             msg += f" Missing permission: {OrganizationManagementLevel.SUPERADMIN}"
             raise PermissionDenied(msg)
 
-        meeting_ids = self.get_all_meeting_ids()
-        errors: Dict[int, str] = {}
-        for meeting_id in meeting_ids:
-            export = export_meeting(self.datastore, meeting_id)
-            checker = Checker(
-                data=export,
-                mode="internal",
-                repair=True,
-                fields_to_remove={
-                    "motion": [
-                        "origin_id",
-                        "derived_motion_ids",
-                        "all_origin_id",
-                        "all_derived_motion_ids",
-                    ]
-                },
-            )
-            try:
-                checker.run_check()
-            except CheckException as ce:
-                errors[meeting_id] = str(ce)
+        export = self.get_all_database()
+        checker = Checker(
+            data=export,
+            mode="all",
+        )
+        errors: str = ""
+        try:
+            checker.run_check()
+        except CheckException as ce:
+            errors = str(ce)
         if not errors:
             return {"ok": True, "errors": ""}
-        return {"ok": False, "errors": self.gen_error_message(errors)}
+        return {"ok": False, "errors": errors}
+
+    def get_all_database(self) -> Dict[str, Any]:
+        export: Dict[str, Any] = {"_migration_index": get_backend_migration_index()}
+        for collection in ALL_COLLECTIONS:
+            export[collection] = remove_meta_fields(
+                self.datastore.get_all(collection, [], lock_result=False)
+            )
+        return export
```

Comment:
You can use the `get_everything` method of the datastore to fetch all data. It is currently only available in dev mode, so we can either make the presenter also only available in dev mode or remove this restriction, but we can think about this later, maybe with @r-peschke. Also, the backend adapter of the DS does not know this route yet, since we did not need it yet. You will have to implement it there, but that should be fairly straight forward.

Reply:
Okay, perhaps improve this later.

Reply:
I use get_everything from the datastore in check_datastore_all